pom.xml

```diff
@@ -82,7 +82,7 @@
 
     <sonar.version>9.4.0.54424</sonar.version>
     <sonarlint.plugin.api.version>6.3.0.36253</sonarlint.plugin.api.version>
-    <analyzer.commons.version>1.24.0.986</analyzer.commons.version>
+    <analyzer.commons.version>1.24.0.990</analyzer.commons.version>
```

Comment:
TODO: change this version to the master one.

Reply:
`1.24.0.991`